Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallDiffCommand.cs

```diff
@@ -472,40 +473,35 @@ private void AddDirectory(string dirName, CancellationToken cancellationToken)
                 throw new FilePathTooLongException(string.Format("Cannot install file {0}, the destination path length has exceeded Windows path length limit (260).", filePath)); 
             }
 #endif
-            string nameHash;
-            if (_mapHashExtractedFiles.TryGetHash(fileName, out nameHash))
-            {
-                var sourceFilePath = Path.Combine(packageDirPath, nameHash + suffix);
-                _logger.LogTrace("sourceFilePath = " + sourceFilePath);
-
-                if (!File.Exists(sourceFilePath))
-                {
-                    throw new MissingFileFromPackageException(string.Format("Cannot find file {0} in diff package.",
-                        fileName));
-                }
-                
-                _logger.LogDebug("Creating file parent directories in local data...");
-                var fileParentDirPath = Path.GetDirectoryName(filePath);
-                _logger.LogTrace("fileParentDirPath = " + fileParentDirPath);
-                //TODO: Assert that fileParentDirPath is not null
-                // ReSharper disable once AssignNullToNotNullAttribute
-                DirectoryOperations.CreateDirectory(fileParentDirPath, cancellationToken);
-                _logger.LogDebug("File parent directories created in local data.");
-
-                _logger.LogDebug("Copying file to local data (overwriting if needed)...");
-                FileOperations.Copy(sourceFilePath, filePath, true, cancellationToken);
-                _logger.LogDebug("File copied to local data.");
-
-                _localMetaData.RegisterEntry(fileName, _versionId, 
-                    _contentSummary.Files.First(x => x.Path == fileName).Size, 
-                    fileIndex == _diffSummary.AddedFiles.Length - 1);
-
-                _logger.LogDebug("Add file entry processed.");
-            }
-            else
+            string nameHash = _mapHashExtractedFiles.GetNameHash(fileName);
+            var sourceFilePath = Path.Combine(packageDirPath, nameHash + suffix);
```

Comment:
do not use `var` is resulting type is unclear

Reply:
Done